gradio/renderable.py

```diff
@@ -76,8 +79,8 @@ def apply(self, *args, **kwargs):
 
 @document()
 def render(
-    inputs: list[Component] | None = None,
-    triggers: list[EventListener] | EventListener | None = None,
+    inputs: list[Component] | Component | None = None,
+    triggers: Sequence[EventListenerCallable] | EventListenerCallable | None = None,
```

Comment:
The only issue with putting `Sequence` here is that it'll affect the docs for gr.render(): https://www.gradio.app/docs/gradio/render

Reply:
And I think many Python users don't know what `Sequence` is

Reply:
Can we add a step in the docs generation to convert `Sequence` container types to `list` @aliabd?

Reply:
I think I can edit `01_render.svx` to do this but maybe we do this once users complain? I would prefer to not introduce a difference between docs and code unless it's absolutely necessary imo.

Reply:
Ok I think this would be applicable in many other parts of the codebase as well (e.g. anywhere we currently type hint list[Component] should really be Sequence[Component]) but doing it later is fine

Reply:
Ok I think we can edit the ParamTable svelte component in that case but will hold off for now.